docs/quickstart.rst

```diff
@@ -90,8 +90,9 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     export BORG_PASSCOMMAND='pass show backup'
 
     # some helpers and error handling:
-    function info  () { echo -e "\n"`date` $@"\n" >&2; }
-    trap "echo `date` Backup interrupted >&2; exit 2" SIGINT SIGTERM
+    info() { printf "\n%s %s\n\n" "$( date )" "$*" >&2; }
```

Comment:
oh, ok.

I am not much into shell scripting. If it is more than some trivial lines, I usually tend to rather use Python.